joplin/settings.py

```diff
@@ -287,24 +297,31 @@
     }
 
     # Specifying the location of files
-    STATICFILES_LOCATION = 'static'
+    if ISSTAGING:
+        STATICFILES_LOCATION = 'static/staging'
+    if ISPRODUCTION:
+        STATICFILES_LOCATION = 'static/production'
+    if ISREVIEWAPP:
+        STATICFILES_LOCATION = 'static/reviewapp'
+    if ISREVIEWAPP:
+        STATICFILES_LOCATION = 'static/local'
```

Comment:
So would all review apps share the same styles? If that's what this implementation demands, then maybe it would make sense to just have Joplin host the styles, rather than S3. That way the styles would be unique to each deployment.

Reply:
This is not too different from how it’s currently set up, but the webpack bundle folder does have versioned hashes.

Also like I mentioned above, an obvious next step would be to make a subfolder based on the branch name or something like that.